packages/babel-parser/ast/spec.md

```diff
@@ -239,6 +239,7 @@ interface NumericLiteral <: Literal {
 ```js
 interface Program <: Node {
   type: "Program";
+  interpreter: string | null;
```

Comment:
The current implementation isn't a string, it's an `InterpreterDirective` node with a `value` string.

Reply:
true, thanks for pointing that out